clustermesh/clustermesh.go

```diff
@@ -140,9 +140,9 @@ func (k *K8sClusterMesh) generateService() (*corev1.Service, error) {
 			svc.Spec.Type = corev1.ServiceTypeLoadBalancer
 			svc.ObjectMeta.Annotations["service.beta.kubernetes.io/azure-load-balancer-internal"] = "true"
 		case k8s.KindEKS:
-			k.Log("🔮 Auto-exposing service within AWS VPC (service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0")
+			k.Log("🔮 Auto-exposing service within AWS VPC (service.beta.kubernetes.io/aws-load-balancer-scheme: internal")
```

Comment:
To match the prior behavior, it looks like we would want to set the new value to "internet-facing", rather than "internal"

Reply:
Bingo! you're right. Fixed